### ISSUE TITLE
Côte d'Ivoire number formatting

### ISSUE THREAD

**Issue by zakhdar** [jackocnr/intl-tel-input#1234]
Hi, 
The numbering in Côte d'Ivoire (Ivory Coast) has changed from 8 digits to 10 digits.
 Would it be possible to update the number format in the library, the new format is as follows: +225 XX XX XX XX XX.

Best regards

**Comment by ccarcamo86** [comment]
Another instance of this issue can be found here: 

https://secure.helpscout.net/conversation/1544814411/1289509?folderId=753362

**Comment by KrohomKrohom** [comment]
Hello, I've also encountered this issue with Côte d'Ivoire phone numbers. They went from 8 digits to 10 digits.
Updating it would be nice. Thanks!

**Comment by jackocnr** [comment]
We use [libphonenumber](https://github.com/googlei18n/libphonenumber) for formatting, validation and generating placeholder numbers - if your issue relates to one of these things, please check [their test site](https://rawgit.com/googlei18n/libphonenumber/master/javascript/i18n/phonenumbers/demo-compiled.html) first and if you see the problem there please file an issue with them instead.

If it's working correctly in libphonenumber, the next step is to check that we're using [the latest version](https://github.com/google/libphonenumber/releases) of libphonenumber in the plugin - check the version history [here](https://github.com/jackocnr/intl-tel-input/commits/master/build/js/utils.js). If we need to upgrade, it only takes 5 minutes to do this yourself and submit a Pull Request - instructions here: [Updating to a new version of libphonenumber](https://github.com/jackocnr/intl-tel-input/blob/master/.github/CONTRIBUTING.md#updating-to-a-new-version-of-libphonenumber).

**Comment by nicolus** [comment]
@jackocnr : I tested it on their site, it is indeed handled correctly, it gives me :

```
****Validation Results:****
Result from isPossibleNumber(): false
Result from isPossibleNumberWithReason(): TOO_SHORT
```

Our version is indeed out of date, I was about to update it but I don't have a bazelisk nor JDK nor ant installed, I don't have a Mac so I can't use brew... so it'll take me more time than I can dedicate to a bug that's not a blocker and only affects a few of our users :-/

@jackocnr I just sponsored you for $20. I don't want my name or company name anywhere, but I'd really appreciate if you could take 5 minutes to update libphonenumber.

**Comment by jackocnr** [comment]
I've just released a new version (v17.0.14) which is built against the latest version of libphonenumber, which should fix the issue. Please let me know if you still see any issues after updating.

**Comment by nicolus** [comment]
@jackocnr : Awesome, this works perfectly and closed 3 long standing tickets in our projects.

Thank you !
